Makefile

```diff
@@ -53,6 +53,9 @@ KUBEBUILDER := $(TOOLS_BIN_DIR)/kubebuilder
 KUSTOMIZE := $(TOOLS_BIN_DIR)/kustomize
 ENVSUBST_BIN := envsubst
 ENVSUBST := $(TOOLS_BIN_DIR)/$(ENVSUBST_BIN)-drone
+SETUP_ENVTEST = $(TOOLS_BIN_DIR)/setup-envtest
+
+ENVTEST_K8S_VERSION := 1.23.x
```

Comment:
I believe we could bump this to v1.25.x considering https://github.com/metal3-io/cluster-api-provider-metal3/pull/755
is on the way but I am fine to keep it as and take care of it in 755

Reply:
I'll just bump it here while I'm at it.